Resources/config/authorize.xml

```diff
@@ -20,7 +20,21 @@
 
         <service id="fos_oauth_server.authorize.form.handler.default" class="FOS\OAuthServerBundle\Form\Handler\AuthorizeFormHandler">
             <argument type="service" id="fos_oauth_server.authorize.form" />
-            <argument type="service" id="request_stack" on-invalid="ignore" />
+            <argument type="service" id="request_stack" />
```

Comment:
I'm not exactly sure what @alcaeus meant here. Should the removal of `on-invalid="ignore"` be reverted?